unifi.js

```diff
@@ -586,6 +586,19 @@ var Controller = function(hostname, port)
     _self._request('/api/s/<SITE>/list/user', null, sites, cb);
   };
 
+  /**
+   * Update device settings, base (using REST)
+   * -----------------------------------------
+   * required paramater <sites>           = name or array of site names
+   * required parameter <device_id>       = 24 char string; _id of the device which can be found with the getAccessDevices() function
+   * required parameter <device_settings> = object containing the configuration to apply to the device, must be a
+   *                                        (partial) object structured in the same manner as is returned by getAccessDevices() for the devic$
+   */
+  _self.setDeviceSettingsBase = function(sites, device_id, deviceSettings)
```

Comment:
Please add a callback `cb` like all other methods have so that this function ends up having `function(sites, device_id, deviceSettings, cb)` and the `_request()` call forwards this callback instead of specifying it as null.